integration_tests/e2e/overview/postInductionOverview.cy.ts

```diff
@@ -132,6 +132,6 @@ context('Prisoner Overview page - Post Induction', () => {
     overviewPage.clickAddGoalButton()
 
     // Then
-    Page.verifyOnPage(CreateGoalPage)
+    Page.verifyOnPage(CreateGoalsPage)
```

Comment:
By clicking the Add Goal button on the Overview page, we arrive on the new Create Goals page, which has a different page class (CreateGoalsPage) (plural)